azure-pipelines.nuget.yml

```diff
@@ -0,0 +1,143 @@
+trigger: none
+
+parameters:
+  - name: version
+    displayName: Version to stamp on binaries and nuget package

```

Comment:
Include an example of the format expected by this parameter.  I don't know if `displayName` is the right field for that, but if there is not another one to use then it would be fine.

Reply:
There doesn't appear to be a better field for this, I'll put it in the displayName. Only other fields are "default" and "values"